network/src/commonTest/kotlin/com/wire/kalium/model/QualifiedSendMessageRequestJson.kt

```diff
@@ -90,7 +90,7 @@ object QualifiedSendMessageRequestJson {
 
     val validDefaultParameters = ValidJsonProvider(
         MessageApi.Parameters.QualifiedDefaultParameters(
-            sender = "sender-client-it",
+            sender = USER_1_CLIENT_1,
```

Comment:
Our old `decodeHexToLong()` was happily swallowing this non hex string but `hexToLong()` throws an exception.